modules/fundamental/src/sbom/service/sbom.rs

```diff
@@ -139,6 +139,10 @@ impl SbomService {
                     .alias("sbom_node", "r0")
                     .translator(|f, op, v| match f.split_once(':') {
                         Some(("label", key)) => Some(format!("labels:{key}{op}{v}")),
+                        None => match f {
```

Comment:
I think it should be possible adding a test for this as well.

Reply:
@ctron, well the following might be simpler :

```suggestion
                    .translator(|f, op, v| match f.split_once(':') {
                        Some(("label", key)) => Some(format!("labels:{key}{op}{v}")),
                        Some(("version", _key)) => Some(format!("versions{op}{v}")),
                        _ => None,
                    }),
```